caps/management/commands/import_filter_data.py

```diff
@@ -0,0 +1,69 @@
+import os
+import pandas as pd
+
+import requests
+import urllib3
+
+import ssl
+
+ssl._create_default_https_context = ssl._create_unverified_context
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+
+from os.path import join
+
+from caps.models import Council, PlanDocument
+
+FILTER_DATA_URL = "https://raw.githubusercontent.com/mysociety/climate_league_filter_datasets/main/data/outputs/all_filters.csv"
+FILTER_CSV = join(settings.DATA_DIR, "all_filters.csv")
+
+
+def get_data():
+    token = os.environ.get("PERSONAL_ACCESS_TOKEN", None)
+    headers = {"Authorization": "token " + token}
+    r = requests.get(FILTER_DATA_URL, headers=headers)
+    with open(FILTER_CSV, "wb") as outfile:
+        outfile.write(r.content)
+
+
+def import_filters():
+    ruc_lookup = dict((desc, code) for code, desc in Council.RUC_TYPES)
+
+    updates = []
+    df = pd.read_csv(FILTER_CSV)
+    for index, row in df.iterrows():
+        council = Council.objects.get(authority_code=row["local-authority-code"])
+
+        quintile = PlanDocument.integer_from_text(row["council-quintile"])
+        if quintile is None:
+            quintile = 0
+
+        try:
+            cluster = ruc_lookup[row["ruc_cluster"]]
+        except KeyError:
+            cluster = ""
+
+        council.population = PlanDocument.char_from_text(row["pop_bucket"])
```

Comment:
Similarly `row["pop_bucket"] = row["pop_bucket"].fillna("")` would be clearer about what was wrong with the column in the first place.